### ISSUE TITLE
TypeError: Cannot read properties of undefined (reading 'startsWith')

### ISSUE THREAD

**Issue by andreas-soroko** [astahmer/openapi-zod-client#45]
Huhu,

getting the following error on  `openapi-zod-client <file> -a --export-schemas --group-strategy 'tag'`. 
(any group-strategy seems to fail)

```
if (schemaName.startsWith("z.")) return;
                             ^

TypeError: Cannot read properties of undefined (reading 'startsWith')
    at addDependencyIfNeeded (node_modules/openapi-zod-client/dist/generateZodClientFromOpenAPI-8e9ca210.cjs.dev.js:1967:24)
    at node_modules/openapi-zod-client/dist/generateZodClientFromOpenAPI-8e9ca210.cjs.dev.js:1972:16
    at Array.forEach (<anonymous>)
```

[for the same payload as here](https://github.com/astahmer/openapi-zod-client/issues/44)

**Comment by astahmer** [comment]
ok got it, looking into it right now !

**Comment by astahmer** [comment]
fixed & published in 1.4.2

**Comment by andreas-soroko** [comment]
fixed.
